src/blocks/backlight.rs

```diff
@@ -234,16 +292,27 @@ impl ConfigBlock for Backlight {
             Some(path) => BacklitDevice::from_device(path, block_config.root_scaling),
             None => BacklitDevice::default(block_config.root_scaling),
         }?;
-
         let brightness_file = device.brightness_file();
 
-        let backlight = Backlight {
+        let (minimum, maximum) = if block_config.minimum <= block_config.maximum {
+            (block_config.minimum, block_config.maximum)
+        } else {
+            (block_config.maximum, block_config.minimum)
+        };
+
+        let backlight = Self {
             id,
             device,
-            step_width: block_config.step_width,
+            step_width: block_config.step_width.max(1),
```

Comment:
Why `.max(1)`? Shouldn't `step_width = 0` disable scrolling?

Reply:
That's a good question. There was probably a reason I though of, but I can't remember. I'll check.

Reply:
Nope, looks like a mistake on my part.
Possibly left over from a first attempt at guaranteeing `brightness > 0` when clamping brightness between `step_width` and `100`. Definitely no use now.